lib/systems/parse.nix

```diff
@@ -294,6 +294,7 @@ rec {
     openbsd  = { execFormat = elf;     families = { inherit bsd; }; };
     solaris  = { execFormat = elf;     families = { }; };
     wasi     = { execFormat = wasm;    families = { }; };
+    emscripten     = { execFormat = wasm;    families = { }; };
```

Comment:
```suggestion
    emscripten = { execFormat = wasm;    families = { }; };
```

Reply:
Best would be to adjust the alignment of the others to match.